index/segment/mem/terms_dict_test.go

```diff
@@ -0,0 +1,170 @@
+package mem
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/m3db/m3ninx/index/segment"
+
+	"github.com/m3db/m3ninx/doc"
+)
+
+var (
+	benchDocs []doc.Document
+)
+
+func init() {
+	var err error
+	if benchDocs, err = readDocuments("../../../testdata/node_exporter.json", 2000); err != nil {
```

Comment:
suggestion(take/leave): iirc this init will be invoked for all test runs (not just benchmark executions). might be better to do this as the first step of the Benchmark... func to avoid the extra time for all unit tests, could use something like https://blog.golang.org/subtests#TOC_6. to make it more manageable

Reply:
Good point, I'll set them as subtests so we can perform the initialization only when these benchmarks are run.